persistence/sql_base_repository.go

```diff
@@ -144,6 +144,9 @@ func (r sqlRepository) queryAll(sq Sqlizer, response interface{}) error {
 	if err != nil {
 		return err
 	}
+	if _, ok := r.sortMappings["random-true"]; ok {
```

Comment:
This should not be here, it is only related to albums, so should be in `album_repository`